lib/db/mysql/patches/patch-000-001.sql

```diff
@@ -0,0 +1,9 @@
+-- Create the 'dbMetadata' table.
+-- Note: This should be the only thing in this initial patch.
+
+CREATE TABLE dbMetadata (
+  name VARCHAR(255) NOT NULL PRIMARY KEY,
+  value VARCHAR(255) NOT NULL
+) ENGINE=InnoDB;
+
+INSERT INTO dbMetadata SET name = 'schema-patch-level', value = '1';
```

Comment:
The patcher currently sanity-checks that the patch correctly updated the level, but yeah, we should iterate on this